modules/subscriptions.php

```diff
@@ -1122,3 +1130,10 @@ function jetpack_do_subscription_form( $instance ) {
 	$output = ob_get_clean();
 	return $output;
 }
+
+jetpack_register_block( 'subscriptions', array( 'render_callback' => 'jetpack_show_subscription_block' ) );
+
+function jetpack_show_subscription_block( $attr, $content ) {
+	$attr['show_only_email_and_button'] = true;
```

Comment:
Instead of doing this can we just pass this attribute via the block save method?